src/main/java/uk/gov/companieshouse/service/dissolution/DissolutionPatcher.java

```diff
@@ -58,6 +62,14 @@ public void updatePaymentInformation(String paymentReference, Timestamp paidAt,
         setDissolutionStatus(dissolution, ApplicationStatus.PAID);
 
         this.repository.save(dissolution);
+
+        final String dissolutionReferenceNumber = dissolution.getData().getApplication().getReference();
+        final String companyName = dissolution.getCompany().getName();
+        final String applicantEmailAddress = dissolution.getCreatedBy().getEmail();
+
+        dissolutionEmailService.sendSuccessfulPaymentEmail(
+            dissolutionReferenceNumber, companyNumber, companyName, applicantEmailAddress
+        );
```

Comment:
Yeah I'd imagine so. We don't want to send an email confirming the application has been paid/received, and then it fails when actually saving the application, so that's why I thought to do it last.